src/themes/jekyll/_includes/components/sw-include.html

```diff
@@ -0,0 +1,14 @@
+<script>
+  if ('serviceWorker' in navigator) {
+    window.addEventListener('load', () => {
+      navigator.serviceWorker.register('sw.js');
+    });
+  } else {
+    console.warn('Your browser does not support service workers.');
```

Comment:
Does this display a message to the window? Developers will try this in Safari and a generic message would be useful for this and other examples.